src/components/Questions/Body.vue

```diff
@@ -246,6 +230,12 @@ export default defineComponent({
       state.isImageLoading = true;
     }
 
+    function checkCharLimit(event: KeyboardEvent) {
+      // checks if character limit is reached in case it is set
+      if (!hasCharLimit.value) return;
```

Comment:
Return for a function called `checkCharLimit` should be `true` or `false` and the caller should decide what to do with this info. Here, this function is deciding what to do, which may not be good in a case where there are multiple use cases for a character limit